### PR TITLE
Remove uses of randomness.

### PR DESCRIPTION
**Pull request by chrisr3**
Remove random UUID generation.
Remove generation of random encryption keys.
Remove unused references to ThreadLocalRandom.
Remove Collections.shuffle() and hardcode default seed for java.util.Random().
Hardcode default seed for java.util.concurrent.ThreadLocalRandom().
Remove native libpack

### CODE REVIEW

File: jdk/src/share/classes/java/math/BitSieve.java [hunk 1]
```diff
@@ -1,212 +0,0 @@
-/*
```
Comment (exFalso) [inline jdk/src/share/classes/java/math/BitSieve.java]: Why remove this? Perhaps just remove the retrieve method?

File: jdk/src/share/classes/java/util/Random.java [hunk 2]
```diff
@@ -98,28 +98,13 @@
...
     /**
-     * Creates a new random number generator. This constructor sets
-     * the seed of the random number generator to a value very likely
-     * to be distinct from any other invocation of this constructor.
+     * Creates a new random number generator. Determinism
+     * requires that we always use the same seed.
      */
     public Random() {
-        this(seedUniquifier() ^ System.nanoTime());
+        this(8006694155008225895L);
```
Comment (exFalso) [inline jdk/src/share/classes/java/util/Random.java]: Where does this number come from?

File: jdk/src/share/classes/java/util/SplittableRandom.java [hunk 3]
```diff
@@ -1,991 +0,0 @@
-/*
```
Comment (exFalso) [inline jdk/src/share/classes/java/util/SplittableRandom.java]: We can keep this class, just make the  seed deterministic

File: jdk/src/share/classes/java/util/concurrent/ThreadLocalRandom.java [hunk 4]
```diff
@@ -133,22 +133,7 @@
...
-            byte[] seedBytes = java.security.SecureRandom.getSeed(8);
-            long s = (long)(seedBytes[0]) & 0xffL;
-            for (int i = 1; i < 8; ++i)
-                s = (s << 8) | ((long)(seedBytes[i]) & 0xffL);
-            return s;
-        }
-        return (mix64(System.currentTimeMillis()) ^
-                mix64(System.nanoTime()));
-    }
+    private static final AtomicLong seeder = new AtomicLong(8270523877573429709L);
```
Comment (exFalso) [inline jdk/src/share/classes/java/util/concurrent/ThreadLocalRandom.java]: Where does this number come from?

**Review by exFalso — COMMENTED** [review]
Couple of things

File: jdk/src/share/classes/java/util/Random.java (hunk 2 above)
Comment (chrisr3) [reply jdk/src/share/classes/java/util/Random.java]: I generated it one day using the above algorithm, and then hardcoded it.

File: jdk/src/share/classes/java/util/concurrent/ThreadLocalRandom.java (hunk 4 above)
Comment (chrisr3) [reply jdk/src/share/classes/java/util/concurrent/ThreadLocalRandom.java]: Pulled it out of my... system clock one afternoon :-).

File: jdk/src/share/classes/java/util/Random.java [hunk 5]
```diff
@@ -98,28 +98,13 @@
...
     /**
-     * Creates a new random number generator. This constructor sets
-     * the seed of the random number generator to a value very likely
-     * to be distinct from any other invocation of this constructor.
+     * Creates a new random number generator. Determinism
+     * requires that we always use the same seed.
      */
     public Random() {
-        this(seedUniquifier() ^ System.nanoTime());
+        this(3141592653589793238L); // Nothing up my sleeve.
```
Comment (exFalso) [inline jdk/src/share/classes/java/util/Random.java]: Perhaps comment about pi instead

File: jdk/src/share/classes/java/util/concurrent/ThreadLocalRandom.java [hunk 6]
```diff
@@ -131,24 +131,9 @@
...
-            byte[] seedBytes = java.security.SecureRandom.getSeed(8);
-            long s = (long)(seedBytes[0]) & 0xffL;
-            for (int i = 1; i < 8; ++i)
-                s = (s << 8) | ((long)(seedBytes[i]) & 0xffL);
-            return s;
-        }
-        return (mix64(System.currentTimeMillis()) ^
-                mix64(System.nanoTime()));
-    }
+    private static final AtomicLong seeder = new AtomicLong(2718281828459045235L);
```
Comment (exFalso) [inline jdk/src/share/classes/java/util/concurrent/ThreadLocalRandom.java:137]: Perhaps comment about e instead

**Review by exFalso — COMMENTED** [review]
Tiny thing

**Review by exFalso — APPROVED** [review]
LGTM 👍

**Pull request merged**